packages/mobile/src/screens/smart-collection-screen/SmartCollectionScreen.tsx

```diff
@@ -33,48 +35,52 @@ const useStyles = makeStyles(({ spacing }) => ({
 }))
 
 type SmartCollectionScreenProps = {
-  smartCollection: SmartCollectionsmartCollection
+  smartCollection: SmartCollection
 }
 
 /**
  * `SmartCollectionScreen` displays the details of a smart collection
  */
-export const SmartCollectionScreen = ({
-  smartCollection
-}: SmartCollectionScreenProps) => {
+export const SmartCollectionScreen = (props: SmartCollectionScreenProps) => {
+  const { smartCollection } = props
+  const { variant } = smartCollection
   const styles = useStyles()
-  const dispatchWeb = useDispatchWeb()
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    dispatch(fetchSmartCollection({ variant }))
+  }, [dispatch, variant])
 
-  const collection = useSelectorWeb((state) =>
+  const collection = useSelector((state) =>
     getCollection(state, { variant: smartCollection.variant })
   )
 
   const playlistName = collection?.playlist_name ?? smartCollection.title
   const description = collection?.description ?? smartCollection.description
 
-  const playlistLibrary = useSelectorWeb(getPlaylistLibrary)
+  const playlistLibrary = useSelector(getPlaylistLibrary)
 
   const isSaved = playlistLibrary
     ? !!findInPlaylistLibrary(playlistLibrary, smartCollection.variant)
     : false
 
   const handlePressSave = useCallback(() => {
     if (collection?.has_current_user_saved) {
-      dispatchWeb(
+      dispatch(
```

Comment:
mind confirming this works?

https://linear.app/audius/issue/C-831/[qa]-explore-playlists-not-saving-in-mobile

Reply:
so guessing this won't work in this branch since none of the social sagas have been added. thinking we address that in a social actions pass soon?

Reply:
Ah yes, sounds good